docs/resources/Audit_Log.md

```diff
@@ -179,6 +183,10 @@ Whenever an admin action is performed on the API, an entry is added to the respe
 | asset                         | [sticker](#DOCS_RESOURCES_STICKER/sticker-object)                                                                           | string                                                                               | empty string                                                                                                                                            |
 | available                     | [sticker](#DOCS_RESOURCES_STICKER/sticker-object)                                                                           | boolean                                                                              | availability of sticker changed                                                                                                                         |
 | guild_id                      | [sticker](#DOCS_RESOURCES_STICKER/sticker-object)                                                                           | snowflake                                                                            | guild sticker is in changed                                                                                                                             |
+| archived                      | [thread](#DOCS_RESOURCES_CHANNEL/thread-metadata-object)                                                                           | bool                                                                            | whether the thread was archived                                                                                                                             |
+| locked                        | [thread](#DOCS_RESOURCES_CHANNEL/thread-metadata-object)                                                                           | bool                                                                            | whether the thread was locked                                                                                                                               |
+| auto_archive_duration         | [thread](#DOCS_RESOURCES_CHANNEL/thread-metadata-object)                                                                           | integer                                                                         | auto archive duration changed                                                                                                                               |
+| type                          | [thread](#DOCS_RESOURCES_CHANNEL/channel-object)                                                                                   | integer  ([channel type](#DOCS_RESOURCES_CHANNEL/#channel-object-channel-types))| channel type of thread changed                                                                                       |
```

Comment:
This can't be changed
This is likely just part of the thread_create event telling you what the initial type was

Reply:
So we use https://github.com/discord/discord-api-docs/pull/3495/files#diff-471f5507a282ec7288c9d909e4e28f65198e4074426cd492a7893a9af4bc9c39R175 ?

Reply:
(the existing type)